### ISSUE TITLE
How to Ban Users ?(Violence action)

### ISSUE THREAD

**Issue by white-devil-00** [EverythingSuckz/TG-FileStreamBot#70]
Can you add feature to ban users who are abusing bot, and for forwarding non appropriate contents. Can you add this ?

**Comment by wiz64** [comment]
> Can you add feature to ban users who are abusing bot, and for forwarding non appropriate contents. Can you add this ?

yes, its possible and easy, the bot can forward user id's along with the files and you may prepare a blacklist command that would put the user id in the blacklist (like /ban 1223431)
or a simple method is to force your users to join a main channel and when they send a file , check if they are on the main...
To ban them, simply ban them off the channel.

The bot can have a small plugin system as a lot of users are asking for extra functionalities, currently a Do-It-Yourself as the code is flexible and easy to modify. I can implement such things if requested.

**Comment by qiangweihewu** [comment]
Yes, would like to see the updates! Thanks!
